pom.xml

```diff
@@ -70,7 +70,7 @@
         <dependency>
             <groupId>org.folio</groupId>
             <artifactId>mod-data-export-spring</artifactId>
-            <version>1.2.0</version>
```

Comment:
Why do you use snapshot version here? Using snapshot versions should be only in exceptional cases.

Reply:
Fixed, thank you.